x-pack/plugins/synthetics/public/apps/synthetics/components/test_run_details/components/test_run_error_info.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiCallOut, EuiText } from '@elastic/eui';
+import * as React from 'react';
+import { SyntheticsJourneyApiResponse } from '../../../../../../common/runtime_types';
+import { StdErrorLogs } from '../../common/components/stderr_logs';
+import {
+  ERROR_RUNNING_TEST,
+  FAILED_TO_RUN,
+} from '../../test_now_mode/browser/browser_test_results';
+
+export const TestRunErrorInfo = ({
+  journeyDetails,
+}: {
+  journeyDetails: SyntheticsJourneyApiResponse['details'];
+}) => {
+  const isDownMonitor = journeyDetails?.journey?.monitor?.status === 'down';
+
+  const errorMessage = journeyDetails?.journey?.error?.message;
+
+  const isStepsLoadingFailed = false;
+
+  return (
+    <>
+      {(isStepsLoadingFailed || isDownMonitor) && (
+        <EuiCallOut
+          data-test-subj="monitorTestRunErrorCallout"
+          title={ERROR_RUNNING_TEST}
+          color="danger"
+          iconType="alert"
+        >
+          <EuiText color="danger">{errorMessage ?? FAILED_TO_RUN}</EuiText>
+        </EuiCallOut>
+      )}
```

Comment:
I would include an `EuiSpacer` here. The content looks a bit too close together in the screenshot.